README.md

```diff
@@ -560,3 +561,25 @@ configure({
   files: ['test/*.ts']
 })
 ```
+
+## Coverage
+You can use [nyc](https://www.npmjs.com/package/nyc) to check the coverage of your code. It works for Typescript files too.
+
+```
+npm install -D nyc
+```
+
+Following is a screenshot from one of my projects.
+
+<img width="789" alt="84738609-da0f2600-afc7-11ea-841c-09273cb29a56" src="https://user-images.githubusercontent.com/1658644/114153565-ba166d80-991f-11eb-8860-426610573c9b.png">
```

Comment:
Where is this image coming from?

Reply:
Your issue 😁 https://github.com/thetutlage/japa/issues/19

Reply:
Can we download the file and keep it inside the repo, so that the file is always present?

Reply:
Should be good now